public/components/overview/office-panel/config/search-bar-config.ts

```diff
@@ -11,17 +11,36 @@
  * Find more information about this on the LICENSE file.
  */
 
-export const filtersValues: { type: string; key: string }[] = [
+export const filtersValues: {
+  type: string;
+  key: string;
+  placeholder: string;
+  optionTitle?: boolean;
+}[] = [
   {
     type: 'combobox',
-    key: 'agent.id',
+    key: 'data.office365.Actor.ID',
+    placeholder: 'Actor ID',
   },
   {
     type: 'combobox',
-    key: 'agent.name',
+    key: 'data.office365.Subscription',
+    placeholder: 'Subscription',
   },
   {
     type: 'combobox',
-    key: 'agent.ip',
+    key: 'data.office365.UserType',
+    placeholder: 'UserType',
```

Comment:
**issue**: The `placeholder` should be readable by the user.

**suggestion**: Maybe we could use `User type`.